wasmdome/src/main.rs

```diff
@@ -30,18 +38,46 @@ enum WasmdomeAction {
     /// Configure local credentials to compete in the arena by submitting an access token
     Compete {
```

Comment:
The compete option for the wasmdome command line is no longer used. We're using the NATS connection and connecting anonymously to that, so the burden of authentication falls on the leaf node, not the wasmdome CLI.